jraft-core/src/main/java/com/alipay/sofa/jraft/storage/impl/LogManagerImpl.java

```diff
@@ -894,6 +894,19 @@ public LogId getLastLogId(final boolean isFlush) {
             Thread.currentThread().interrupt();
             throw new IllegalStateException(e);
         }
+        if (c.lastLogId == null) {
+            assert stopped;
```

Comment:
Why not just throw the `IllegalStateException` when the `c.lastLogId` is null? Looks like it's not necessary to wait the `shutdownLatch` and return something in this case.